src/app.js

```diff
@@ -70,6 +77,31 @@ app.get("/api/movies/:id/screenings", async (request, response) => {
   }
 });
 
+app.get("/aboutus", async (request, response) => {
+  renderPage(response, "aboutus");
+});
+
+app.get("/newsevents", async (request, response) => {
+  renderPage(response, "newsevents");
+});
+
+app.post("/api/movies/review", (request, response) => {
+  //Extracts URL-Query string to object
+  const reviewAtributes = reviewParser(request);
```

Comment:
Instead of passing the entire `request`, I would just pass `request.body` (and change the function accordingly). That way, the function doesn't need to _concern_ itself with more than necessary, which is an important aspect of _separation of concerns_.